addons/binding/org.openhab.binding.nibeuplink/src/main/java/org/openhab/binding/nibeuplink/handler/UplinkPolling.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.nibeuplink.handler;
+
+import org.openhab.binding.nibeuplink.internal.command.GenericStatusUpdate;
+import org.openhab.binding.nibeuplink.internal.command.NibeUplinkCommand;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Polling worker class. This is responsible for periodic polling of status values.
+ *
+ * @author Alexander Friese - initial contribution
+ */
+public class UplinkPolling implements Runnable {
+    /**
+     * Logger
+     */
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    /**
+     * Handler for delegation to callbacks.
+     */
+    private final NibeUplinkHandler handler;
+
+    /**
+     * Constructor.
+     *
+     * @param handler
+     * @param config
```

Comment:
`config` is obsolete
also: please either complete the JavaDoc (better) or drop it completely (not so good). This looks unfinished.

Reply:
This applies to a lot of JavaDoc throughout the binding. Please review all others.

Reply:
did a lot of updates on incomplete JavaDoc